mne/io/curry/curry.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+import os
+import re
+import numpy as np
+import mne
+
+from ..base import BaseRaw, _check_update_montage
+from ..meas_info import create_info
+from ..utils import _read_segments_file, warn, _find_channels, _create_chs
+
+
+def _read_curry_events(fname_base, curry_vers):
+    """
+    Read curry files and convert the data for mne.
+    Inspired by Matt Pontifex' EEGlab loadcurry() extension.
+
+    :param full_fname:
+    :return:
+    """
```

Comment:
Make sure to follow our docstrings. Check our contributing guide. If its not clear how we do it, ask for changes in the contributing guide.